modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java

```diff
@@ -48,6 +48,9 @@ public CountryRegionUpgradeProcess(
 
 	@Override
 	protected void doUpgrade() throws Exception {
+		CompanyCountriesUtil.updateRegionCounter(getConnection());
```

Comment:
We should not make these methods public. Updating a counter should an exceptional operation that must be executed internally in the UpgradeProcess

Reply:
I put it public because I need them in the upgrade process and in the CompanyCountriesUtil itself. In this case is better to replicate both methods in the two classes?

Reply:
@smottal Is it really needed in both?

I mean, all customers will execute the upgrade process except the new ones. I guess the new ones won't have issues since there are no region and regionLocalization records.

Reply:
As discussed we keep the methods public